ui/src/components/yaml/index.tsx

```diff
@@ -254,6 +255,8 @@
       setStreaming(false)
     }
```

Comment:
Consider adding a null check for `interpretEndRef.current` before calling `getBoundingClientRect()` to prevent potential runtime errors.